src/eventHandlers/InterviewEventHandler.js

```diff
@@ -33,6 +37,123 @@ async function sendInvitationEmail (payload) {
   })
 }
 
+/**
+ * Check if there is overlapping interview, if there is overlapping, then send notifications.
+ *
+ * @param {Object} payload the event payload
+ * @returns {undefined}
+ */
+async function checkOverlapping (payload) {
+  const interview = payload.value
+  const overlappingInterview = await models.Interview.findAll({
+    where: {
+      [Op.or]: [{
+        startTimestamp: {
+          [Op.lt]: interview.endTimestamp,
+          [Op.gte]: interview.startTimestamp
+        }
+      }, {
+        endTimestamp: {
+          [Op.lte]: interview.endTimestamp,
+          [Op.gt]: interview.startTimestamp
+        }
+      }, {
+        [Op.and]: [{
+          startTimestamp: {
+            [Op.lt]: interview.startTimestamp
+          }
+        }, {
+          endTimestamp: {
+            [Op.gt]: interview.endTimestamp
+          }
+        }]
+      }]
+    }
+  })
```

Comment:
We should **not** take into account interviews in status `Completed` or `Cancelled` for both `interview` and `overlappingInterview`.